sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -238,7 +237,7 @@ Use the `listChatThreads` method to retrieve a list of threads.
 ```swift
 import AzureCore
 let options = ListChatThreadsOptions(maxPageSize: 1)
-client.listChatThreads(withOptions: options) { result, _ in
+chatClient.listChatThreads(withOptions: options) { result, _ in
```

Comment:
If this is a `ChatClient` which always operates on Chats, how about naming this e.g. `chatClient.listThreads()` instead?